lib/tdf3/src/tdf.ts

```diff
@@ -946,7 +947,7 @@ async function updateChunkQueue(
             bufferSize
           );
           if (buffer) {
-            sliceAndDecrypt({
+            await sliceAndDecrypt({
```

Comment:
We dont need await here. Dowload (getPayloadSegment) doesnt need to wait for dowloaded chunk to decrypt to start to dowload next chunk. Its just slowedown file download. 
<img width="1146" alt="image" src="https://github.com/opentdf/client-web/assets/41641307/c21be074-b490-456f-8916-5dd6a93aa8cf">

stream waits for new chunk in chun map, slice and decrypt provide it. Dowload doesnt have to stop.

Reply:
Thanks for the explanation! Updated!